Dockerfile

```diff
@@ -20,4 +20,4 @@ EXPOSE 5100
 
 HEALTHCHECK CMD curl --fail http://localhost:5100 || exit 1 
 
-CMD ["python", "eq_cir_management_ui.app:app", "--host", "0.0.0.0", "--port", "5100"]
+CMD ["python", "app", "--host", "0.0.0.0", "--port", "5100"]
```

Comment:
Since this runs on built-in development server (Uvicorn or Flask's development server) shouldn't we explicitly use a server of our choice? It is simpler but not as robust or performant as Gunicorn for future production use.

Reply:
Will look at adding Gunicorn rather than defaulting